src/Courses/TopicGrades/GradingPage.tsx

```diff
@@ -31,7 +31,7 @@ export const TopicGradingPage: React.FC<TopicGradingPageProps> = () => {
     const [gradeOverride, setGradeOverride] = useState<Partial<StudentGrade>>({});
     const [isPinned, setIsPinned] = useState<Pin | null>(null); // pin one or the other, not both
     const [topic, setTopic] = useState<TopicObject | null>(null);
-    const [problems, setProblems] = useState<ProblemObject[] | null>(null);
+    const [problems, setProblems] = useState<NewProblemObject[] | null>(null);
```

Comment:
We're still using `NewProblemObject`? Does it have something the regular `ProblemObject` doesn't have?

Reply:
extends ProblemObject and includes courseTopicContentId -- which I ended up using as a test to determine whether or not we need to re-fetch our problems and/or re-map those problems...